src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
```

Comment:
Add some modifiers so reviewers can test your solution. Looks like it doesn't work

![image](https://github.com/mate-academy/layout_calendar/assets/63243136/f1d198c3-015e-4045-928e-06424b236629)